src/main/java/org/folio/inventory/dataimport/handlers/actions/InstanceUpdateDelegate.java

```diff
@@ -53,7 +53,7 @@ public Future<Instance> handle(Map<String, String> eventPayload, Record marcReco
       var mappedInstance = recordMapper.mapRecord(parsedRecord, mappingParameters, mappingRules);
       InstanceCollection instanceCollection = storage.getInstanceCollection(context);
 
-      return getInstanceById(instanceId, instanceCollection)
+      return ConsortiumUtil.findInstanceById(instanceId, instanceCollection)
```

Comment:
Seems weird that ConsortiumUtil is used to get an instance even when the tenant is non-ECS. I think `getInstanceById` should be here. Inside `getInstanceById` we would determine if to use the consortiumUtil or the instanceCollection.